docs/runbooks/flux/synthetic-checks.md

```
@@ -0,0 +1,59 @@
+- Jobs:
+  - `synthetic-health`: every 15 minutes (`*/15 * * * *`)
+  - `synthetic-e2e`: every hour (`0 * * * *`)
+- Manual trigger: `workflow_dispatch`
+
+## What It Checks
+
+1. `synthetic-health`
+- Calls `FLUX_HEALTH_URL` (default: `https://uzywajqzbdbrfammshdg.supabase.co/functions/v1/flux-health`)
+- Fails if HTTP request fails or `.ok != true`
+
+2. `synthetic-e2e`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)